waspc/packages/deploy/src/providers/railway/deploy/RwDeployOptions.d.ts

```diff
@@ -0,0 +1,7 @@
+import { CommonOptions } from '../../shared/CommonOptions';
+
+export interface RwDeployOptions extends CommonOptions {
```

Comment:
Do you think at the type-level, using the full name makes sense? e.g swap Rw for Railway?

Reply:
full name wouldn't hurt :)